lib/src/main/java/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -181,28 +192,28 @@ class StatefulLayout : FrameLayout, StateContainer<Int, State> {
      * @throws [NoSuchElementException] if [id] was not found.
      */
     override fun showState(@IdRes id: Int): State {
-        return showState(id, true)
+        return showState(id, areTransitionsEnabled)
     }
 
     /**
      * Show a state
      *
      * @param id
-     * @param playTransition if true, the transition will be played
+     * @param areTransitionEnabled if true, the transition will be played
      * @return
      */
-    fun showState(@IdRes id: Int, playTransition: Boolean): State {
+    fun showState(@IdRes id: Int, areTransitionEnabled: Boolean): State {
```

Comment:
So this parameter bypasses StatefulLayout global parameter without modifying it?
The fact that the function parameter is named after the class parameter may be confusing. You lack the info that you the transitions will be dis/enabled only for this state update

Reply:
Maybe renaming it "isThisTransitionEnabled" is enough